src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Charts/LineChart/SubscaleLineChart/ChartTooltip/ChartTooltip.styles.ts

```diff
@@ -26,11 +28,13 @@ export const StyledMdPreview = styled(MarkDownPreview)`
   font-size: ${variables.font.size.md};
   font-weight: normal;
   text-align: initial;
-  padding: ${theme.spacing(0.7, 0)};
 
   .default-theme {
     p {
       padding: 0;
+      margin: ${theme.spacing(-1.0, 0, 1.6, -0.3)};
```

Comment:
@sultanofcardio what do you think about this -0.3 padding to the left. I applied it to allign the tooltip title with the parragraph as requested by Matt. I tried to set all padding and margins to 0 expecting that the parragraph and title align by not providding any additional style but there was still an small space on the parragraph element that placed it a little more to the right.

Reply:
They align for me without this negative margin. With the margin added (or subtracted in this case) the paragraph is now more to the left than the title

| without margin (0px) | with margin (-3px) |
| --- | --- |
| <img width="1552" alt="image" src="https://github.com/user-attachments/assets/0a851f32-dc22-4eba-bc67-c8b0bdcb3944"> | <img width="1552" alt="image" src="https://github.com/user-attachments/assets/953dc8f5-b105-4c85-9046-6396ae2685c9"> |

I'm not sure why they are out of alignment for you, but I'd stay away from the negative margin. This is already a nitpick and I wouldn't spend too much time on this, but @farmerpaul do you have any idea why this might be happening?

Reply:
Thanks Kenroy, you are right. It kind of looks aligned now after deleteing that negative padding and the negative padding kind of makes it look a little to the right. I Updated my parragrapg style on my latest commit.